src/ui/page/share/dataset/steps/summary/summary.scss

```
@@ -0,0 +1,101 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins';
+
+.okp4-dataverse-portal-share-dataset-summary-container {
+  @include with-theme {
+    @include grid-container($colGutter: 48px, $rowGutter: 22px);
+    grid-template-rows: auto 1fr 1fr;
+    height: 100%;
+    padding: 34px 10px 0 0;
+    box-sizing: border-box;
+    overflow-y: auto;
+
+    @include use-breakpoints(('desktop')) {
+      @include grid-container($colGutter: 34px, $rowGutter: 26px);
+    }
+
+    h2 {
+      @include grid-item($colStart: 1, $colEnd: -1, $rowStart: auto, $rowEnd: auto);
```

```suggestion
      @include grid-item($colStart: 1, $colEnd: -1);
```
The mixin has been updated to replace undefined arguments as `auto`

---

Nice catch, fixed!